pkg/multiTenancyPlugins/authorization/volumes.go

```diff
@@ -40,21 +38,21 @@ func filterVolumes(r *http.Request, rec *httptest.ResponseRecorder) []byte {
 	}
 	return buf.Bytes()
 }
-func hostFSMountCheck(tenantId string, volumeBindings[] string) (error) {
-	for _,b := range volumeBindings {
-		if index := strings.Index(b,":"); index > -1 {
-		  v := b[0:index]
-		  if strings.Contains(v,"/") {
-			return errors.New(HOST_FS_MOUNT_NOT_ALLOWED)
-		  }
+func hostFSMountCheck(tenantId string, volumeBindings []string) error {
+	for _, b := range volumeBindings {
+		if index := strings.Index(b, ":"); index > -1 {
+			v := b[0:index]
+			if strings.Contains(v, "/") {
```

Comment:
Interesting problem:
   There are two parsers, one in volume/volume_unix.go and one in volume/volume_windows.go, with a function ParseMountSpec.   volume/volume_unix.go uses filepath.ToSlash, strings.SplitN, and strings.Count to parse the bind.     volume/volume_windows.go use some complicated regex to do the job.

I have not figured out how the code decides which parser to use ( I think the call happens in daemon/volumes.go registerMountPoints).  However, from the names I'm guessing that when docker is running in unix then volume_unix.go called.

Should a modify the code to do similar parse to volume_unix.go?